Disc/Networking/Helpers/Request.swift

```diff
@@ -115,3 +115,19 @@ func createRequestParameters(scopes scopes: [Scope] = [], provider: Provider, co
         + createRequestParameters(code: code)
         + createRequestParameters(redirectUri: redirectUri)
 }
+
+func createRequestParameters(clientId clientId: String, scopes: [Scope] = [], provider: Provider, token: String, secret: String? = .None) -> [String: String] {
+    return createRequestParameters(clientId: clientId)
+        + createRequestParameters(scopes: scopes)
+        + createRequestParameters(provider: provider)
+        + createRequestParameters(token: token)
+        + createRequestParameters(secret: secret)
+}
+
+func createRequestParameters(clientId clientId: String, scopes: [Scope] = [], provider: Provider, code: String, redirectUri: String) -> [String: String] {
+    return createRequestParameters(clientId: clientId)
+        + createRequestParameters(scopes: scopes)
+        + createRequestParameters(provider: provider)
+        + createRequestParameters(code: code)
+        + createRequestParameters(redirectUri: redirectUri)
+}
```

Comment:
It should be possible to implement these as follows, although maybe they could be refactored to use partial application instead.

Thankfully, these aren't `public` so we can address later.

```swift
func createRequestParameters(clientId clientId: String, scopes: [Scope] = [], provider: Provider, token: String, secret: String? = .None) -> [String: String] {
    return createRequestParameters(clientId: clientId)
        + createRequestParameters(scopes: scopes, provider: provider, token: token, secret: secret)
}

func createRequestParameters(clientId clientId: String, scopes: [Scope] = [], provider: Provider, code: String, redirectUri: String) -> [String: String] {
    return createRequestParameters(clientId: clientId)
        + createRequestParameters(scopes: scopes, provider: provider, code: code, redirectUri: redirectUri)
}
```